apps/docs/pages/guides/auth/sessions.mdx

```diff
@@ -0,0 +1,151 @@
+
+A session terminates, depending on configuration, when:
+
+- The user clicks sign out.
+- The user changes their password or performs a security sensitive action.
+- It times out due to inactivity.
+- It reaches its maximum lifetime.
+
+## Access token (JWT) claims
+
+Every access token contains a `session_id` claim, a UUID, uniquely identifying the session of the user. You can correlate this ID with the primary key of the `auth.sessions` table.
+
+## Sign out and scopes
+
+Supabase Auth allows you to specify three different scopes for when a user invokes the [sign out API](/docs/reference/javascript/auth-signout) in your application:
+
+- Global (default) when all sessions active for the user are terminated.
+- Local, to only terminate the current session for the user but keep sessions on other devices or browsers active.
+- Others, to terminate all but the current session for the user.
+
+You can invoke these by providing the `scope` option:
+
+```typescript
+await supabase.auth.signOut()
+// is equivalent to
+await supabase.auth.signOut({ scope: 'global' })
+```
+
+Use `local` or `others` for the rest.
+
+Upon sign out all refresh tokens and potentially other database objects related to the affected sessions are destroyed.
+
+Be aware that access tokens of revoked sessions remain valid until their expiry time encoded in the `exp` claim.
+
+## Controlling the lifetime of a session
+
+<Admonition type="note">
+
+This feature is only available on Pro plans and up.
+
+</Admonition>
+
+Supabase Auth can be configured to limit the lifetime of a user's session. By default, all sessions are active until the user clicks sign out or performs some other action that terminates a session.
+
+In some applications, it's useful or required from a security point of view to ensure that users authenticate often, or that sessions are not left active on devices for too long.
+
+There are two ways in which you can limit the lifetime of a session:
+
+- By enabling time-boxed sessions which terminate after a fixed amount of time.
+- By enabling inactivity timeout sessions that have not been refreshed for a while will be regarded as terminated.
+
+To make sure that users are required to re-authenticate periodically, you can set a positive value for the _Time-box user sessions_ option in the [Auth settings](/dashboard/project/_/settings/auth) for your project.
+
+To make sure that sessions expire after a period of inactivity, you can set a positive duration for the _Inactivity timeout_ option in the [Auth settings](/dashboard/project/_/settings/auth).
+
+Sessions are not proactively destroyed when you change these settings, but rather the check is enforced whenever a session is refreshed next. Otherwise sessions are progressively cleaned up 24 hours after they expire, which prevents you from causing a high load on your database by accident and allows you some freedom to undo changes without adversely affecting all users.
```

Comment:
```suggestion
Sessions are not proactively destroyed when you change these settings. The check is enforced when the session is next refreshed. Otherwise sessions are progressively cleaned up 24 hours after they expire, which prevents you from causing a high load on your database by accident and allows you some freedom to undo changes without adversely affecting all users.
```

a bit confused about what the `otherwise` is referring to here. what are the two possible cases?

Reply:
Hmm ok I'll re-write it. The idea is that the effect is immediate, but not the changes in the database that lag 24 hours and happen progressively to help them avoid minor mistakes that log everyone out.